### ISSUE TITLE
draw_line thickness regression in 3.x branch

### ISSUE THREAD

**Issue by EIREXE** [godotengine/godot#55920]
### Godot version

3.x (4cae48d81c783f2d124b09e9decf739aad17cbb7)

### System information

Windows 11 and Arch

### Issue description

The behavior of draw_line has changed from the stable 3.3.4 branch, now lines seem to be twice as thick with the same thickness value.

3.3.4 (before):
![](https://i.imgur.com/QnGDUul.png)

3.x (after):
![](https://i.imgur.com/POdKktl.png)

(Note the thickness difference in the lines at the bottom of the screen)

### Steps to reproduce

Run:

```
func _draw():
    draw_line(Vector2(0, 100), Vector2(200, 100), Color.blue, 3)
```

And observe how it's different on different versions of the engine.

**Comment by Calinou** [comment]
@EIREXE Which stretch mode are you using in your project? This will affect line thickness depending on the window size.

**Comment by EIREXE** [comment]
> @EIREXE Which stretch mode are you using in your project? This will affect line thickness depending on the window size.

I am using stretch mode 2d with aspect expand

**Comment by lawnjelly** [comment]
Ah that might be me, I recently faffed with thick line drawing in #54377.

Ah yes I can see it already, the multiplier should probably be half the width. I'll test this and make a PR to fix. :+1:

**Comment by lawnjelly** [comment]
Fixed by #55922 .
